R/mod_view_tables.R

```diff
@@ -103,6 +111,30 @@ mod_view_tables_server <-
       offset = 0
     )
     
+    vars <- reactive({
+      var <- vector()
+      for(column_name in names(table_info$data)){
+        unique_values_number <- length(unique(table_info$data[[column_name]]))
+        if(unique_values_number <= 500){
```

Comment:
What happens if this number is >500? It doesn't get added to datamods tab right?